hazelcast/src/main/java/com/hazelcast/instance/AddressPicker.java

```diff
@@ -17,6 +17,8 @@
 package com.hazelcast.instance;
 
 import com.hazelcast.cluster.Address;
+import com.hazelcast.internal.cluster.impl.MemberHandshake;
+import com.hazelcast.internal.server.tcp.SendMemberHandshakeTask;
```

Comment:
As these are only used in the Javadoc, I wouldn't add imports here. (Rather use FQCN in the Javadoc)